configs/default/HGLR-HGLRCF411SX1280_15A.config

```diff
@@ -0,0 +1,110 @@
+# Betaflight / STM32F411 (S411) 4.4.0 Jan 28 2023 / 23:33:31 (4605309) MSP API: 1.45
+#mcu STM32F411
+
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_ICM42688P
+#define USE_ACCGYRO_BMI270
+#define USE_MAX7456
+#define USE_FLASH_W25Q128FV
+
+board_name HGLRC_F411_SX1280_15A

```

Comment:
ERROR: File name does not match board name (HGLRC_F411_SX1280_15A) / manufacturer id (HGLR) in Unified Target configuration configs/default/HGLR-HGLRCF411SX1280_15A.config.